### PR TITLE
Support `--aux` for vembrane table.

### PR DESCRIPTION
**Pull request by tedil**
This PR adds support for the `--aux name=file` option also available for filter and tag.
It also removes the `--all` option, which I had no idea existed and did not do anything.

### CODE REVIEW

File: vembrane/modules/table.py [hunk 1]
```diff
@@ -39,13 +39,6 @@ def add_subcommmand(subparsers):
         metavar="CHAR",
         help="Define the field separator (default: \\t).",
     )
-    parser.add_argument(
-        "--all",
-        "-a",
-        default=False,
-        action="store_true",
-        help="Do not filter duplicate entries.",
-    )
```
Comment (dlaehnemann) [inline vembrane/modules/table.py:48]: Was this removal intentional?

**Review by dlaehnemann — APPROVED** [review]
Looks good to me. One question in the code, and:

* Does this need a test, or is the respective functionality covered by the other commands, anyway?

File: vembrane/modules/table.py (hunk 1 above)
Comment (tedil) [reply vembrane/modules/table.py:48]: Yes, I was confused that the short options clashed (`-a`), then had a look where `--all` was used. Turns out: nowhere.
